src/main/resources/org/opentripplanner/submodes/submodes.csv

```diff
@@ -0,0 +1,31 @@
+name, mode, description, netexSubmodes, gtfsExtendedRouteTypes
+DOMESTIC, AIRPLANE, "An aeroplane service to a domestic destination.", "[domesticFlight]", "[1102]"
+HELICOPTER, AIRPLANE, "A helicopter service.", "[helicopterService]", "[1110]"
+INTERNATIONAL, AIRPLANE, "An aeroplane service to an international destination.", "[internationalFlight]", "[1101]"
+AIRPORT_LINK, BUS, "A bus route specifically aimed at transporting passengers to and from airports. Usually with special board/alight-rules on the intermediary stops.", "[airportLinkBus]", "[]"
+EXPRESS, BUS, "A regional bus serving fewer stops than its regionalBus counterpart (express departure). Not to be confused with coach services.", "[expressBus, "[702]"
```

Comment:
I have not been able to find a definitive way to specify lists in a CSV file, since it's not part of the specification. As we probably will only have NeTEx and GTFS submode values here for the foreseeable future, it is probably sufficient with a space between values.